Library/Formula/gitlab-ci-multi-runner.rb

```diff
@@ -16,21 +18,35 @@ class GitlabCiMultiRunner < Formula
   end
 
   depends_on "go" => :build
+  depends_on "godep" => :build
 
-  def install
-    ENV["GOPATH"] = buildpath
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+      :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
 
+  def install
     mkdir_p buildpath/"src/gitlab.com/gitlab-org"
     ln_sf buildpath, buildpath/"src/gitlab.com/gitlab-org/gitlab-ci-multi-runner"
 
-    # gitlab-ci-multi-runner's deps is managed by godeps
-    ENV.append_path "GOPATH", "#{buildpath}/Godeps/_workspace"
+    ENV["GOPATH"] = buildpath
+
+    ENV.prepend_create_path "PATH", buildpath/"bin"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd buildpath/"src/github.com/jteeuwen/go-bindata/go-bindata" do
+      system "go", "install"
+    end
 
     cd "src/gitlab.com/gitlab-org/gitlab-ci-multi-runner" do
       commit_sha = `git rev-parse --short HEAD`
 
+      # Disable vender support for go 1.5 and above
+      ENV["GO15VENDOREXPERIMENT"] = "0"
+
       # Copy from Makefile
-      system "go", "build", "-o", "gitlab-ci-multi-runner", "-ldflags", "-X main.NAME=gitlab-ci-multi-runner -X main.VERSION=#{version} -X main.REVISION=#{commit_sha}"
+      system "make", "executors/docker/bindata.go"
```

Comment:
Can you check upstream? If it isn't going to move until the next release we can vendor and checksum it rather than permitting undeclared arbitrary downloads.